src/validators/account-validator.js

```diff
@@ -0,0 +1,231 @@
+    (request, response, next) => {
+        validateResult(request, response, next);
+    }
+]
+
+const validationStatusAccount  = [
+    check('status')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isNumeric().withMessage('El campo no debe ser un número')
+        .matches(/^[1-3]$/).withMessage('Permitir solo números del 1 al 3')
+]
+
+
+const validationCreationAccount = [
+    validationAccount,
+    validationUsername,
+    validationEmail,
+    validationPassword,
+    check('role')
+        .exists().withMessage('El campo debe existir')
+        .isString().withMessage('El campo debe ser una cadena')
+        .isIn(['user', 'manager']).withMessage('El rol no es válido'),
+    check('idCity')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .matches(/^[a-z0-9]{24}$/).withMessage('El ID debe tener números y letras minúsculas')
+        .isLength(24).withMessage('Debe tener 24 caracteres'),
+    (request, response, next) => {
+        validateResult(request, response, next);
+    }
+]
+
+
+const validationId  = [
+    check('_id')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .matches(/^[a-z0-9]{24}$/).withMessage('El ID debe tener números y letras minúsculas')
+        .isLength(24).withMessage('Debe tener 24 caracteres')
+]
+
+const validationUpdateAccount  = [
+    validationId,
+    validationAccount,
+    validationUsername,
+    validationEmail,
+    (request, response, next) => {
+        validateResult(request, response, next);
+    }
+]
+
+const validationChangeStatusAccount = [
+    validationId,
+    validationStatusAccount,
+    (request, response, next) => {
+        validateResult(request, response, next);
+    }
+]
+
+const validationAccountId  = [
+    param('accountID')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .matches(/^[a-z0-9]{24}$/).withMessage('El ID debe tener números y letras minúsculas')
+        .isLength(24).withMessage('Debe tener 24 caracteres'),
+    (request, response, next) => {
+        validateResult(request, response, next);
+    }
+]
+
+const validationAccountFilters  = [
+    param('filter')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isIn(['lastname', 'name', 'age', 'dateBirth', 'email', 'username', 'dateCreation']).withMessage('El filtro es inválido'),
+    param('criterion')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío'),
+    (request, response, next) => {
+        validateResult(request, response, next);
+    }
+]
+
+const validationCriterion = (request, response, next) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const validationCriterion = (request, response, next) => {
```